manual/manual/cmds/intf-c.etex

```diff
@@ -1166,6 +1166,19 @@ It would be incorrect to perform
 has taken place since "r" was allocated.
 
 
+\subsection{Asynchronous callbacks}
+\label{caml-check-urgent-gc}
+
+Since 4.10, allocation functions called from C do not call finalisers,
+signal handlers, and memory profiler callbacks, but delays their
+execution. These asychronous callbacks can execute arbitrary OCaml
+code, including raising asynchronous exceptions.
+
+The function \verb"caml_check_urgent_gc" from "memory.h" checks for
```

Comment:
Thanks for the docs @gadmm. Isn't that a very bad name for what it's doing ? I mean if I'm `check`ing  something I rather expect an existence test, not side effects. Maybe `caml_exec_urgent_gc` would be a better name ?

Reply:
This name cannot change for backwards compat, but I'm open to suggestions for the name of a new function that takes `void` as argument instead. `caml_async_callbacks`, `caml_safe_point`...

Reply:
Ah sorry didn't get this was already in the wild.